src/app/leaves/actions/update-leave-request.test.ts

```diff
@@ -0,0 +1,26 @@
+import { waitFor } from "@testing-library/dom";
+import { LeaveRequestStatus, LeaveTypes } from "@/constant/constant";
+import { updateLeaveRequest } from "./updated-leave-request";
+
+describe("Leave Request Update Test", async () => {
+  const mockPayload = {
+    memberId: 12,
+    leaveType: LeaveTypes.Vacation,
+    startDate: "2025/11/20",
+    endDate: "2025/11/26",
+    reason: "Due to some personal reason",
+    status: LeaveRequestStatus.Pending,
+  };
+  const mockLeaveRequestId = 2;
```

Comment:
Let's just call it `leaveRequestId`. Since we are already in test environment, we already know it's a mock.